sdk/src/main/java/com/horizen/account/secret/PrivateKeySecp256k1Creator.java

```diff
@@ -32,16 +26,9 @@ public static PrivateKeySecp256k1Creator getInstance() {
 
     @Override
     public PrivateKeySecp256k1 generateSecret(byte[] seed) {
-        try {
-            ECKeyPair keyPair = Keys.createEcKeyPair(new SecureRandom(seed));
-            // keyPair private key can be 32 or 33 bytes long
```

Comment:
So how do we manage 33 bytes case now?
I see that in the new code the commented line was removed and another line was added: "// Check why is this failing"